README.md

```diff
@@ -21,12 +21,19 @@ Bare minimum features needed to satisfy use case:
 ### Phase 1: Building a House
 *Setup a server that:*
   - *Serves a basic html file with a WebGL canvas saying "hello" in 3D*
-  - *Exposes a basic GraphQL endpoint returning "world"*
-- [ ] Todo items for this phase, in order
+  - *Exposes a basic GraphQL endpoint returning "hello"*
+- [ ] Create a DJango web server
+- [ ] Initialize a GraphQL endpoint & return some nonsense
+- [ ] Write a thin SDK on the frontend to execute API calls
+- [ ] Successfully incorporate nonsense from API in the rendering
+- [ ] Setup a database solution & store some nonsense in it
+- [ ] Return that nonsense through the API
 
 ### Phase 2: Going Grocery Shopping
 *Collect the necessary data from free resources that would be needed for the MVP (for example, what stars can we see from where)*
-- [ ] Todo items for this phase, in order
+- [ ] Store star catalog information in database
+- [ ] Store bortle location data in database
```

Comment:
is future plan and not MVP, remove?

Reply:
That's correct that we haven't listed Bortle filtration in the MVP. Though I wonder if we should make sure that this dataset exists/is accessible to us before we move forward? Unless we would be okay with making the MVP and then realizing this dataset is inaccessible... What's your perspective on this?